wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java

```diff
@@ -196,16 +189,12 @@ public Response create(
     return Response.status(201).entity(asDtoWithLinksAndToken(workspace)).build();
   }
 
-  @Beta
   @Path("/devfile")
   @POST
-  @Consumes({APPLICATION_JSON, "text/yaml", "text/x-yaml"})
+  @Consumes({APPLICATION_JSON})
```

Comment:
Does this mean we can no longer send YAML files to this endpoint?

Reply:
The answer is yes and no :). You can't send send it directly anymore (like, direct method call). But you still send it via HTTP to endpoint, and it will be intercepted by `DevfileEntityProvider` which will covert yaml to valid DTO which can be consumed by this method.

Reply:
UPD: oh, yes. it does really won't work if media types isn't match. Reverted.